package/clp.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import re
+
+def grab(url_name):
+  """
```

Comment:
I think we can remove this comment and or put it in a shortly written way inside the readme?

Reply:
i'll try to add this comment into the readme, sort of like a short and precise explanation as to what this feature does? would that be better?

Reply:
yes, absolutely